### ISSUE TITLE
New Presets - workout video energy

### ISSUE THREAD

**Issue by VISHWAJ33T** [microfox-ai/mediamake#67]
### JSON Array (Required)

[
    {
        "prompt": "Create a high-intensity transition effects preset that mimics professional sports broadcast transitions with aggressive directional wipes and motion blur. Think of how ESPN transitions between replays - sharp, fast, and impactful. The preset should use multiple overlapping layers that create a 'swoosh' effect, where content slides in from one direction while the previous content slides out with motion blur and scaling. Include energetic shake effects at the transition point to emphasize the impact, similar to how action scenes are cut in high-energy workout montages. The transitions should feel like they're being 'punched' through the screen with explosive energy.",
        "technicalSpecs": "Structure: Use BaseLayout with absolute positioning for layered transitions. Create 3-4 overlay layers using 'absolute inset-0 w-full h-full' for full coverage. Implement directional wipes using clip-path animations or translateX/Y with scale transforms. Use generic keyframe effects with ranges for smooth progression. Add motion blur via CSS filter blur during movement (blur: 0 -> 8px -> 0). Include a shake effect using internal 'shake-impact' preset at 50% timeline mark. Timing: Total duration 0.8-1.2s with aggressive ease-in-out curves. Layer animations should overlap by 0.2s for fluid motion. Performance: Use transform and opacity only, add will-change: transform, filter to animated layers. Consider using GPU-accelerated properties exclusively. For workout context, sync transitions to audio beats using waveform effects with high sensitivity (0.8+) to match workout tempo."
    },
    {
        "prompt": "Design a glitch-style transition effect preset inspired by digital interference and VHS tracking errors, perfect for high-energy workout videos with electronic music. Visualize the effect as if someone is fast-forwarding through a worn VHS tape during an intense training montage. The transition should incorporate RGB channel splitting, horizontal displacement bars that slide across at different speeds, and brief moments of static noise. Add chromatic aberration effects where colors separate and reconverge, creating that classic analog video distortion. Include rapid flickering opacity changes to simulate signal loss, making it feel like the video is struggling to keep up with the workout intensity.",
        "technicalSpecs": "Structure: BaseLayout container with multiple child layers for RGB splitting. Use 'relative w-full h-full overflow-hidden' for main container. Create 3 identical content layers with 'absolute inset-0 mix-blend-screen' for RGB channels. Implement horizontal scan lines using repeated thin BaseLayout strips with 'h-px w-full bg-white opacity-20'. Effects: Use generic keyframe effects for translateX displacement (-20px to 20px randomly). Apply CSS filters for hue-rotate on RGB layers (0deg, 120deg, 240deg). Add brightness and contrast fluctuations (brightness: 100% -> 150% -> 100%). Create static noise with rapidly changing opacity (0 -> 0.3 -> 0) on a noise texture overlay. Timing: Main transition 0.6-0.8s with multiple micro-animations at 0.05s intervals. Use stepped easing for digital feel. Audio sync: Implement waveform effects tied to bass frequencies for glitch intensity, threshold at 0.6 for selective triggering on heavy beats."
    },
    {
        "prompt": "Build a kinetic typography transition preset that treats text and content like explosive particles in a high-impact workout environment. Imagine each element as a weight being thrown with force - content should burst apart into fragments that scatter outward with physics-based motion, then rapidly reassemble for the incoming content. Think of it like a shattered mirror reforming in reverse, where each shard has its own trajectory and rotation. The effect should convey raw power and energy, as if the transitions themselves are part of the workout, with elements moving like they have mass and momentum.",
        "technicalSpecs": "Structure: Use BaseLayout with grid layout 'grid grid-cols-4 grid-rows-4 gap-0' to divide content into 16 fragments. Each fragment gets 'relative overflow-hidden' with transform-origin variations. Apply staggered animations using repeatChildrenProps with index-based delays. Effects: Implement explosion using scale (1 -> 0.8 -> 1.2 -> 0) and translate with radial directions calculated from center. Add rotation animations (0deg -> 720deg) with different speeds per fragment. Use opacity fade (1 -> 0.6 -> 0) for dispersal effect. Incoming content uses reverse animation with spring easing. Performance: Use transform-style: preserve-3d on container, individual transforms on fragments. Batch DOM updates by using CSS custom properties for dynamic values. Timing: Total 1s duration, explosion phase 0.4s with ease-out, reassembly 0.6s with spring easing. Audio reactivity: Use waveform 'scale' effect on fragments, sensitivity 0.7, reacting to mid-range frequencies for punch."
    },
    {
        "prompt": "Create a liquid metal morphing transition effect that flows like mercury between scenes, perfect for showcasing transformation in fitness content. Envision the transition as molten chrome that ripples and reshapes, similar to the T-1000 effect but applied to video transitions. The content should appear to liquefy at the edges first, creating organic flowing shapes that merge and separate with surface tension effects. Add metallic shine highlights that move across the surface during the morph, emphasizing the three-dimensional quality of the transformation. The effect should feel both futuristic and organic, representing the body's adaptation during intense training.",
        "technicalSpecs": "Structure: BaseLayout with SVG filter effects for liquid distortion. Main container uses 'relative w-full h-full' with overflow-hidden. Apply SVG turbulence filter with animated baseFrequency. Create metallic shine using gradient overlays with 'absolute inset-0 bg-gradient-to-br from-white/30 via-transparent to-white/20'. Effects: Use CSS filter with url(#liquid-filter) referencing SVG feTurbulence and feDisplacementMap. Animate filter properties via generic effects (baseFrequency: 0.01 -> 0.05 -> 0.01). Add specular highlights using moving linear gradients with transform translateX animation. Implement edge liquefaction with clip-path using complex path animations. Include subtle color shifts using hue-rotate for chrome effect. Timing: 1.2s total, liquefaction starts at edges (0-0.4s), center morphs (0.4-0.8s), solidification (0.8-1.2s). Use custom cubic-bezier(0.45, 0.05, 0.55, 0.95) for organic flow. Performance: Isolate filter effects to specific layers, use CSS containment for optimization."
    },
    {
        "prompt": "Develop a strobe-flash transition preset that simulates high-speed photography captures during explosive athletic movements. Picture a series of rapid-fire camera flashes that freeze moments in time, creating a stroboscopic effect where you see multiple positions of the movement simultaneously. Each flash should overexpose the image to pure white for a split second, then reveal the next frame with high contrast. Add motion trails that persist briefly between flashes, showing the path of movement like long-exposure sports photography. The effect should feel like watching someone train under a disco strobe light at maximum intensity.",
        "technicalSpecs": "Structure: BaseLayout with multiple snapshot layers using 'absolute inset-0 pointer-events-none'. Create flash overlay with 'fixed inset-0 bg-white z-50' for full-screen flash. Motion trail layers use 'absolute inset-0 opacity-60' with decreasing opacity. Effects: Flash effect using opacity animation (0 -> 1 -> 0) over 0.05s with linear easing. Implement exposure burst with CSS filter brightness(100% -> 300% -> 100%). Create motion trails by duplicating content with staggered opacity and slight transform offsets. Add contrast pump using filter contrast(100% -> 200% -> 100%) synchronized with flashes. Timing: Total 0.8s with 5-6 flash intervals at 0.15s each. Motion trails persist for 0.3s with exponential decay. Audio sync: Use waveform 'opacity' effect on flash layer, triggered by high-frequency peaks (threshold 0.7). Beat detection for flash timing using audio analysis API, selecting impactful beats only."
    },
    {
        "prompt": "Design a torn paper rip transition that feels like aggressively tearing through workout plans to reveal the next exercise. Visualize someone ripping through paper with raw strength - the tear should be jagged and violent, with paper fibers visible at the edges. The rip should start from multiple points simultaneously, as if grabbed by multiple hands, creating an organic tearing pattern. Add subtle paper texture shadows and depth to make it feel three-dimensional. Include small paper fragments that flutter away from the tear, adding kinetic energy to the transition. The effect should convey breaking through limitations.",
        "technicalSpecs": "Structure: Use BaseLayout with CSS mask-image for tear shape. Create tear mask using SVG path with jagged edges or CSS polygon clip-path. Layer structure: background content, tear mask layer, foreground content, floating fragments. Apply 'relative w-full h-full' to container with 'preserve-3d' transform-style. Effects: Animate clip-path or mask-position to reveal tear progressively. Add box-shadow for depth '0 4px 20px rgba(0,0,0,0.3)' on tear edges. Create paper texture using CSS filter with slight noise/grain. Floating fragments use transform rotate3d with physics-based animation. Include subtle skew transform on torn edges for 3D effect. Timing: Tear initiation 0.2s (multiple start points), main tear 0.4s with ease-out, fragments float 0.8s total. Use staggered delays for organic feel. Performance: Pre-calculate clip-path points, use CSS custom properties for dynamic values. GPU acceleration via transform and opacity only for fragments."
    },
    {
        "prompt": "Create an electric surge transition preset that channels raw energy through the screen like lightning conducting through a workout machine. Imagine electricity arcing between scenes with bright plasma effects, creating branching patterns like neural networks firing during intense exercise. The transition should include flickering electric blue edges, sparks that scatter from connection points, and a brief electromagnetic distortion that warps the content. Add pulsing glow effects that radiate outward from the energy discharge points, making it feel like the video itself is powered by the workout's intensity.",
        "technicalSpecs": "Structure: BaseLayout with Canvas or SVG for lightning generation. Main container 'relative w-full h-full bg-black' for contrast. Create lightning paths using SVG path with stroke animations. Spark particles as absolute positioned divs with 'w-1 h-1 bg-blue-400 rounded-full'. Effects: Generate lightning with SVG path stroke-dasharray animation. Add glow using CSS filter drop-shadow(0 0 10px rgba(59, 130, 246, 0.8)). Implement electromagnetic warp with transform skew and scale oscillations. Create energy pulse with radial gradient animations expanding outward. Flicker effect using rapid opacity changes (1 -> 0.3 -> 1) at 0.02s intervals. Timing: Build-up 0.2s, main discharge 0.3s, afterglow 0.3s (0.8s total). Use step timing for flicker, spring easing for warps. Audio integration: Waveform 'brightness' effect synced to bass for glow intensity. Trigger sparks on beat detection with high impact threshold. Performance: Use CSS animation over JS, batch spark creation, limit particle count to 20-30."
    },
    {
        "prompt": "Build a shatter-and-reform glass transition effect where content appears to break like a mirror hit by a medicine ball, then reconstructs for the incoming scene. Envision safety glass cracking in a spider web pattern from multiple impact points, with fragments catching light as they separate and rotate in 3D space. Each shard should have realistic physics - larger pieces fall slower, smaller ones scatter further. Add prismatic light refraction effects as fragments move, creating rainbow edges. The reformation should feel like time reversing, with fragments magnetically pulling back together with satisfying precision.",
        "technicalSpecs": "Structure: BaseLayout with Voronoi tessellation for realistic crack patterns. Container uses 'relative w-full h-full perspective-1000' for 3D depth. Generate 30-50 fragments using calculated polygons with 'absolute clip-path-polygon'. Each fragment gets unique transform-origin based on position. Effects: Shatter uses transform rotateX/Y/Z with physics-based values (larger fragments: ±30deg, smaller: ±180deg). Add translateZ for depth separation (-100px to 300px based on size). Implement refraction with hue-rotate on edges during movement. Glass shine effect using linear-gradient overlays with opacity animation. Drop shadow for depth '0 10px 40px rgba(0,0,0,0.4)' on fragments. Timing: Impact moment 0.1s, shatter expansion 0.5s ease-out, hang time 0.2s, reform 0.6s ease-in-out. Stagger fragments by distance from impact point. Audio: Use waveform 'shake' effect at impact moment, high sensitivity for dramatic effect."
    },
    {
        "prompt": "Design a portal vortex transition that pulls content into a swirling vortex like being sucked into a high-intensity training zone. Visualize a spiraling whirlpool effect where the current scene gets twisted and stretched into the center, while the new scene emerges from the same point spinning outward. The vortex should have multiple rotation layers creating a tunnel effect, with motion blur increasing toward the center. Add particle effects that spiral along the vortex path like sweat droplets in a centrifuge. The transition should feel like entering another dimension of fitness intensity.",
        "technicalSpecs": "Structure: BaseLayout with nested rotating containers for vortex layers. Main container 'relative w-full h-full overflow-hidden perspective-800'. Create 5-6 concentric layers with 'absolute inset-0' and increasing scale. Particle system using absolutely positioned elements following spiral paths. Effects: Rotate animation with increasing speed per layer (innermost: 720deg, outermost: 180deg). Scale transform from 1 to 0 for suction, 0 to 1 for emergence. Apply spiral distortion using CSS transform rotate3d and translateZ. Motion blur via filter blur (0px -> 15px at center -> 0px). Particle paths using animation along SVG spiral path or calculated positions. Timing: Suction phase 0.6s with ease-in, emergence 0.6s with ease-out (1.2s total). Layer animations offset by 0.1s for smooth spiral. Performance: Use transform-only animations, CSS will-change on all rotating elements. Limit particles to 15-20 for smooth 60fps. Audio sync: Waveform 'rotate' effect matching vortex speed to beat intensity."
    },
    {
        "prompt": "Create a thermal heat map transition effect that visualizes the energy and intensity of a workout through color temperature changes. Imagine an infrared camera switching between normal and thermal vision - hot zones pulse with reds and oranges while cooler areas fade to blues and purples. The transition should show heat signatures blooming outward from high-activity areas, with thermal waves rippling across the screen. Add heat distortion effects like you'd see above hot pavement, making the image shimmer and warp. Include particle effects representing heat dissipation, like steam rising from intense exercise zones.",
        "technicalSpecs": "Structure: BaseLayout with multiple blend mode layers for thermal effect. Base container 'relative w-full h-full bg-gradient-radial from-red-600 to-blue-900'. Thermal overlay using 'absolute inset-0 mix-blend-multiply opacity-80'. Heat distortion layer with animated turbulence filter. Particle system for steam effects using transform translateY animations. Effects: Color mapping using CSS filter hue-rotate animation (0deg -> 180deg -> 360deg). Heat bloom with radial gradients expanding from hot spots. Shimmer distortion using SVG feTurbulence with animated seed values. Implement thermal waves with concentric ring animations using scale and opacity. Steam particles rise with translateY and fade with opacity (1 -> 0 over 2s). Timing: Color shift 0.4s, heat bloom 0.6s overlapping, distortion throughout 1s. Use sine wave easing for organic thermal flow. Audio reactivity: Map audio intensity to heat zones using waveform 'color' effect. Higher frequencies trigger hotter colors, bass adds distortion intensity. Performance: Composite thermal layers on GPU, limit particles, use CSS containment."
    }
]
